lib/new_relic/agent/configuration/manager.rb

```diff
@@ -154,8 +158,21 @@ def apply_transformations(key, value)
           end
         end
 
+        def enforce_allowlist(key, value)
+          return unless allowlist = default_source.allowlist_for(key)
+          return if allowlist.include?(value)
+
+          default = default_source.default_for(key)
+          ::NewRelic::Agent.logger.warn "Invalid value '#{value}' for #{key}, applying default value of '#{default}'"
```

Comment:
I know you're matching the syntax for the other references in this file, but I don't think we need the `::` before `NewRelic` if we get around to cleaning this up in the future.

Reply:
Addressed with [642ca83](https://github.com/newrelic/newrelic-ruby-agent/pull/2546/commits/642ca83eaf4b4a6812180eacab39675df7044c72)